web/app/tournament/tourny.py

```diff
@@ -169,3 +172,93 @@ def select_player(tournament_id, tournament_player_id):
         # flash("Could not delete player: {{ tp.name }}")
         return redirect(url_for('.players_index',
             tournament_id=tournament_id))
+
+
+@tournament.route('/<tournament_id>/matches/', methods=['GET', 'POST'])
+def matches_index(tournament_id):
```

Comment:
I like this. I think this is better. I don't have much comfort with ajax. I would be happy to wrestle with it. Having it under my belt would, I think, allow for all resources on a url (GET, POST, PUT, DELETE) to get their own methods. Right now I'm sneaking PUT and DELETE under the guise of POST. Flask probably has various ways of doing this already but...